dags.yaml

```diff
@@ -177,7 +177,7 @@ bqetl_subplat:
 
   default_args:
     owner: srose@mozilla.com
-    start_date: "2021-07-20"
+    start_date: "2025-04-17"
```

Comment:
Changing the start date along with the schedule change, so if catchup is ever enabled on this DAG without simultaneously resetting the start date, then Airflow won't try to rerun four years of DAG runs.